lua/luasnip/util/path.lua

```diff
@@ -51,18 +51,15 @@ function Path.read_file(path)
 	return buf
 end
 
-local MYCONFIG_ROOT = vim.env.MYVIMRC
--- if MYVIMRC is not set then it means nvim was called with -u
--- therefore the first script is the configuration
--- in case of calling -u NONE the plugin won't be loaded so we don't
--- have to handle that
+local MYCONFIG_ROOT = vim.env.MYCONFIG_ROOT
```

Comment:
This is here so it can be overridden from the outside?

Reply:
Yes, if you don't like it I can remove it, I thought it was what you meant in your last comment.

Reply:
Ohhh, no I didn't mean that. Though I can see I didn't word that all that well.
I don't think we need that mechanism.